framework/docs/source/how-to-use-json-output.rst

```diff
@@ -0,0 +1,166 @@
+Use JSON outputs for Flower CLIs
+================================
+
+The `Flower CLIs <ref-api-cli.html>`_ come with a built-in JSON output mode. This mode
+is useful when you want to consume the output of a Flower CLI programmatically. For
+example, you might want to use the output of the ``flwr`` CLI in a script or a
+continuous integration pipeline.
+
+To enable JSON output, simply pass the ``--format json`` flag to the CLI. In this guide,
+we'll show you how to use JSON output with the ``flwr run``, ``ls``, and ``stop``
+commands.
+
+.. |flwr_run| replace:: ``flwr run``
+
+.. |flwr_ls| replace:: ``flwr ls``
+
+.. |flwr_stop| replace:: ``flwr stop``
+
+.. _flwr_ls: ref-api-cli.html#flwr-ls
+
+.. _flwr_run: ref-api-cli.html#flwr-run
+
+.. _flwr_stop: ref-api-cli.html#flwr-stop
+
+``flwr run`` JSON output
+------------------------
+
+The |flwr_run|_ command runs a Flower app. By default, the command prints the status of
+the app build and run process as follows:
+
+.. code-block:: bash
+
+    $ flwr run
```

Comment:
I wonder if there is value in adding a comment saying that the below assumes you are in the directory of the app (same applies to other commands)

Reply:
Added, thanks.